docs/charts.md

```diff
@@ -441,6 +413,41 @@ myimports:
 
 The parent's final values now contains the `myint` and `mybool` fields imported from subchart1.
 
+### Managing Dependencies manually via the `charts/` directory
+
+If more control over dependencies is desired, these dependencies can
+be expressed explicitly by copying the dependency charts into the
+`charts/` directory.
+
+A dependency can be either a chart archive (`foo-1.2.3.tgz`) or an
+unpacked chart directory. But its name cannot start with `_` or `.`.
+Such files are ignored by the chart loader.
+
+For example, if the WordPress chart depends on the Apache chart, the
+Apache chart (of the correct version) is supplied in the WordPress
+chart's `charts/` directory:
+
+```
+wordpress:
+  Chart.yaml
+  requirements.yaml
+  # ...
+  charts/
+    apache/
+      Chart.yaml
+      # ...
+    mysql/
+      Chart.yaml
+      # ...
+```
+
+The example above shows how the WordPress chart expresses its dependency
+on Apache and MySQL by including those charts inside of its `charts/`
+directory.
+
+**TIP:** _To drop a dependency into your `charts/` directory, use the
+`helm fetch` command or use a `requirements.yaml` file_
```

Comment:
Because `requirements.yaml` is covered above, it should probably be removed from here. If someone is manually populating dependencies, they should just use `helm fetch` or copy manually